Python 2 seems to be an unlisted dependency when doing the initial 'npm install'

When I ran the initial `npm install`, the process threw errors 

```
C:\Users\Andrew\Documents\ScreepsTypescript\screeps-typescript-starter>npm install

> @screeps/driver@3.2.1 install C:\Users\Andrew\Documents\ScreepsTypescript\screeps-typescript-starter\node_modules\@screeps\driver
> node-gyp rebuild -C native


C:\Users\Andrew\Documents\ScreepsTypescript\screeps-typescript-starter\node_modules\@screeps\driver>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild -C native )  else (node "C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js" rebuild -C native )
gyp ERR! configure error
gyp ERR! stack Error: Can't find Python executable "python", you can set the PYTHON env variable.
gyp ERR! stack     at PythonFinder.failNoPython (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\configure.js:484:19)
gyp ERR! stack     at PythonFinder.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\configure.js:509:16)
gyp ERR! stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\graceful-fs\polyfills.js:284:29
gyp ERR! stack     at FSReqWrap.oncomplete (fs.js:153:21)
gyp ERR! System Windows_NT 10.0.17134
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild" "-C" "native"
gyp ERR! cwd C:\Users\Andrew\Documents\ScreepsTypescript\screeps-typescript-starter\node_modules\@screeps\driver\native
gyp ERR! node -v v10.15.3
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @screeps/driver@3.2.1 install: `node-gyp rebuild -C native`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @screeps/driver@3.2.1 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Andrew\AppData\Roaming\npm-cache\_logs\2019-03-25T18_04_11_296Z-debug.log
```

Installing Python 3.4 produced different errors (Abbreviated)

```
gyp ERR! configure error
gyp ERR! stack Error: Command failed: C:\Python34\python.EXE -c import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack   File "<string>", line 1
gyp ERR! stack     import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack                                ^
gyp ERR! stack SyntaxError: invalid syntax
```

--- a/README.md
+++ b/README.md
@@ -8,7 +8,10 @@ You will need:
 
  - [Node.JS](https://nodejs.org/en/download) (>= 8.0.0)
  - A Package Manager ([Yarn](https://yarnpkg.com/en/docs/getting-started) or [npm](https://docs.npmjs.com/getting-started/installing-node))
- - Rollup CLI (Optional, install via `npm install -g rollup`)
+ - Rollup CLI (Optional, install via `npm install -g rollup`) 
+ - Python 2 (for node-gyp, [Python 3 is not supported](https://github.com/nodejs/node-gyp/issues/193))
+ - Build tools (`apt install build-essential` for Ubuntu, [Visual Studio](https://www.visualstudio.com/vs/) for Windows, etc) 
+
 
 Download the latest source [here](https://github.com/screepers/screeps-typescript-starter/archive/master.zip) and extract it to a folder.
 

